pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundleFactory.java

```diff
@@ -204,6 +207,25 @@ public NamespaceBundle getBundlesWithHighestTopics(NamespaceName nsname) {
         });
     }
 
+    public NamespaceBundle getBundleWithHighestThroughput(NamespaceName nsName) {
+        LoadManager loadManager = pulsar.getLoadManager().get();
+        if (loadManager instanceof ModularLoadManagerWrapper) {
```

Comment:
`instanceof`  is usually a code smell, but I cannot find a better way to suggest